audit_exceptions/gnome_devel_allowlist.json

```diff
@@ -1,4 +1,5 @@
 {
+  "gtk-doc": "1.33",
```

Comment:
From livecheck comment:
```
  # We use a common regex because gtk-doc doesn't use GNOME's "even-numbered
  # minor is stable" version scheme.
```
1.33.2 is current latest in distros like:
- Fedora: https://src.fedoraproject.org/rpms/gtk-doc
- Ubuntu 21.04: https://packages.ubuntu.com/hirsute/gtk-doc-tools
- ArchLinux: https://archlinux.org/packages/extra/any/gtk-doc/
- Gentoo: https://packages.gentoo.org/packages/dev-util/gtk-doc
- ...